doc/specs/stdlib_math.md

```diff
@@ -320,7 +320,7 @@ program demo_logspace_rstart_cbase
 
 end program demo_logspace_rstart_cbase
 ```
-### `arange`
+### `arange` - Creates fixed-spaced values of given spacing, within a given interval
```

Comment:
Adding a description here breakes references to this function from the docs. Maybe just as separate line afterwards to keep the URL short?

Reply:
I'm not sure if such a description needs to be created for every procedure, but it does make the URL longer.

Or I just describe: "`arange` function"? Just like "`clip` function".